test/parallel/test-release-changelog.js

```diff
@@ -0,0 +1,89 @@
+'use strict';
+
+// This test checks that the changelogs contain an entry for releases.
+
+const common = require('../common');
+const assert = require('assert');
+const fs = require('fs');
+const path = require('path');
+
+const getDefine = (text, name) => {
+  const regexp = new RegExp(`#define\\s+${name}\\s+(.*)`);
+  const match = regexp.exec(text);
+  assert.notStrictEqual(match, null);
+  return match[1];
+};
+
+const srcRoot = path.join(__dirname, '..', '..');
```

Comment:
I think this will fail when testing the build https://github.com/nodejs/node/blob/main/doc/contributing/releases.md#10-test-the-build

Reply:
@RafaelGSS It shouldn't matter. The test doesn't depend on the version of Node.js running the test -- instead it is checking consistency between `src/node_version.h`, `CHANGELOG.md` and `doc/changelogs/CHANGELOG_V<major ver>.md`. 

The idea is that for a release proposal (i.e. the release commit) this new test would fail if there is an inconsistency (e.g. the main `CHANGELOG.md` wasn't updated (e.g. https://github.com/nodejs/node/pull/45269#issuecomment-1314121623 🙂)).

For https://github.com/nodejs/node/blob/main/doc/contributing/releases.md#10-test-the-build if using the proposal branch you're likely to be either on the release commit (in which case the test runs and checks consistency, but the test should already have run and passed for the proposal PR) or on the "Working on" commit in which case the test will be skipped (or will fail if you update the version but don't reset the is_release bit).

Reply:
Alright